metadata-app/src/components/RequiredMetadata/index.js

```diff
@@ -212,35 +230,27 @@ const RequiredMetadata = (props) => {
         />
       </div>
       <div className="grid-row margin-top-3">
-        <WrappedField
-          label="Publisher"
-          name="publisher"
-          type="select"
-          choices={leafPublishers}
-          required
-          className="error-msg"
-          helptext={helpTexts.select}
-          infoText="The publishing entity (e.g. your agency) and optionally their parent organization(s)."
-          errors={errors}
-        />
-        <WrappedField
-          name="publisher_other"
-          type="string"
-          helptext={helpTextify(
-            `If you selected “Other”, please specify the name of your Publisher`
-          )}
-          disabled={values.publisher !== 'Other'}
-          errors={errors}
-          required
-        />
-      </div>
-      <div className="grid-row margin-top-3">
-        <WrappedField
-          label="Sub Agency (optional)"
-          name="subagency"
-          type="string"
-          errors={errors}
-        />
+        <div className="grid-col-12">
+          <span className="usa-label">
+            Select Publisher
+            <ToolTip>
+              <h3>Publisher</h3>
+              <p>The publishing entity (e.g. your agency, department, bureau, or office).</p>
+            </ToolTip>
+          </span>
+          <Autocomplete
+            name="publisher"
+            type="string"
+            value={values.publisher}
+            // TODO - inputValue should be replaced with parent dataset name
```

Comment:
This is a leftover

Reply:
@kmanaseryan not super clear but would it still work as expected? Or is it unfinished feature?

Reply:
@anuveyatsu I meant the TODO comment is leftover, it came when I've reused the other autocomplete snippet, so don't pay attention on that line.